frontend/src/components/common/Invite.vue

```diff
@@ -60,150 +51,194 @@ const objectPermCodes: Record<string, string> = {
   UPDATE: 'Update'
 };
 
-const formData: Ref<any> = ref({
-  expiresAt: 86400,
-  isRestricted: false,
-  permCodes: ['READ'],
-});
-
 // Permissions selection
 const selectedOptions = computed(() => {
   return props.resourceType === 'bucket' ? bucketPermCodes : objectPermCodes;
 });
 
-const isOptionUnselectable = (optionName: string) => {
-  // Make default permission disabled
-  return optionName === 'Read';
-};
+// Form validation schema
+const schema = yup.object().shape({
+  email: yup
+    .string()
+    .matches(new RegExp(Regex.EMAIL), 'Provide a valid email address')
+    .when('emailType', {
+      is: (string: string) => string === 'single',
+      then: (schema) => schema.required('Email address is required')
+    }),
+  multiEmail: yup
+    .array()
+    .transform(function (value, originalValue) {
+      if (this.isType(value) && value !== null) return value;
+      return originalValue ? originalValue.split(/[\r\n ,;]+/).filter((item: string) => item) : [];
```

Comment:
It splits by `\n` in addition to `,`/`;`? I think we should also include that in the error message.

Reply:
Yeah maybe. Tyler wanted to require comma or semicolon. so that's the preference.
but i think it could be useful to accept newlines as well which would allow for example, pasting from a spreadsheeet or a list in another document.
Whether we want to mention new lines in the error message or placeholder would be a question for Tyler.